### ISSUE TITLE
Replace Pylama to Pylava

### ISSUE THREAD

**Issue by dunossauro** [dunossauro/splitty#12]
Pylama doesn't work on new versions of python. Replace pylama to pylava on tox.

**Comment by dunossauro** [comment]
How to solve that:

In tox.ini file we have pylama
```ini
[testenv:lint]
deps =
  coverage
  pep8
  pep257
  pyflakes
  pylama
  xenon

commands =
  coverage run --source=splitty -m unittest discover -s tests/
  xenon --max-absolute B --max-modules A --max-average A splitty
  pylama splitty/
```

Only replace pylama to pylava and run tests.

**Comment by Surya-06** [comment]
I shall work on this issue.

**Comment by dunossauro** [comment]
Fixed in #15

:heart_decoration:
